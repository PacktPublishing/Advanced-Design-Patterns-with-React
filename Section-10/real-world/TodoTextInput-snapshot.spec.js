import React from 'react'
import renderer from 'react-test-renderer'
import TodoTextInput from './TodoTextInput'

test('snapshots are awesome', () => {
  const component = renderer.create(<TodoTextInput onSave={() => {}} />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
