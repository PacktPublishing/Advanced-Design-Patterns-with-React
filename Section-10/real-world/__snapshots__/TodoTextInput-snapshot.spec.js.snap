exports[`test snapshots are awesome 1`] = `
<input
  autoFocus="true"
  className=""
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  placeholder={undefined}
  type="text"
  value="" />
`;
